docs/user_guides/single_table/constraints.rst

```diff
@@ -200,7 +228,8 @@ constraints that we just defined as a ``list``:
     constraints = [
         unique_company_department_constraint,
         age_gt_age_when_joined_constraint,
-        years_in_the_company_constraint
+        years_in_the_company_constraint,
+        reasonable_age_constraint
```

Comment:
Are there any problems that arise from having to constraints on the age column?

Reply:
I didn't notice any.